python/postprocessing/modules/jme/jetmetHelperRun2.py

```diff
@@ -7,10 +7,13 @@
 from PhysicsTools.NanoAODTools.postprocessing.modules.jme.fatJetUncertainties import *
 
 # JEC dict
+# https://twiki.cern.ch/twiki/bin/viewauth/CMS/JECDataMC#Recommended_for_MC
 jecTagsMC = {
     '2016': 'Summer16_07Aug2017_V11_MC',
     '2017': 'Fall17_17Nov2017_V32_MC',
     '2018': 'Autumn18_V19_MC',
+    'UL2016_preVFB': 'Summer19UL16APV_V7_MC',
```

Comment:
Can the name be changed to preVFP please?

Reply:
Done. My bad.